### DISCUSSION TITLE
Its been 3 years since the first Bluetooth Icon request, can we have it please

### DISCUSSION THREAD

**Discussion by TrustyTechSG** [tailwindlabs/heroicons#996] [Ideas]
![image](https://github.com/tailwindlabs/heroicons/assets/36871034/a0f81471-2dd2-44c0-95cc-bb722f578d1d)

![image](https://github.com/tailwindlabs/heroicons/assets/36871034/63fe7221-9b8f-4e40-b692-59bc4f9db0af)

**Comment by zefir-git** [comment]
Why burden Heroicons with brands/logos when https://simpleicons.org/ is specifically for this purpose
